modules/layouts/plugins/access/civicrm_checksum_layout_access.inc

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * @file
+ * Plugin to provide access if two user account contexts are equal.
```

Comment:
@laryn Can you make this file comment more relevant? Something like "Plugin to provide access if a CiviCRM contact checksum is valid, or if a user is logged in."

Reply:
Thank you!